bom/application/pom.xml

```diff
@@ -1194,6 +1194,11 @@
                 <artifactId>quarkus-flyway-db2-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-liquibase-common</artifactId>
+                <version>${project.version}</version>
+            </dependency>
```

Comment:
You don't think it's worth making it a proper extension from the get go?

Asking because maybe there are some things we could share in the processors too (I wouldn't do that in this patch but that's a reason why having a proper extension might make sense).

Reply:
> You don't think it's worth making it a proper extension from the get go?

It could go either way so I opted to not do it until we really need it.